scripts/change_url

```diff
@@ -107,8 +107,12 @@ fi
 # UPDATE THE DATABASE
 #=================================================
 
-ynh_mysql_execute_as_root --sql="UPDATE wp_options SET option_value='$new_domain$new_path' WHERE option_name='siteurl'" --database=$app
-ynh_mysql_execute_as_root --sql="UPDATE wp_options SET option_value='$new_domain$new_path' WHERE option_name='home'" --database=$app
+# Get the database table prefix
+db_prefix=$(grep '^$table_prefix' "$final_path/wp-config.php" | sed "s/.*'\(.*\)'.*/\1/" )
+ynh_debug --message="db_prefix=$db_prefix"
```

Comment:
Is it part of the PR or it was for the related issue #67?

Reply:
Yep, it was only part of the issue.
I'll remove that.